init.lua

```diff
@@ -2,7 +2,7 @@
 -- TODO: Check for terrain height
 
 -- Defines the edge of a world
-local edge = tonumber(minetest.setting_get("world_edge")) or 30000
+local edge = tonumber(minetest.settings:get("world_edge")) or 30000
```

Comment:
May not want to merge this until after 0.4.16 release, as the new ['settings' object calls][l_settings] are not available in 0.4.15.

[l_settings]: https://github.com/minetest/minetest/blob/43d1f375d18a2fbc547a9b4f23d1354d645856ca/src/script/lua_api/l_settings.cpp#L267